.travis.yml

```diff
@@ -6,6 +6,4 @@ cache:
 language: java
 script: "mvn verify"
 jdk:
-  - openjdk7
```

Comment:
Why were these Travis JDK's removed?

Reply:
After merging the "owlapi_authorization" branch into "owlcs/owlapi:version4" branch, Maven pom file build plugin compiler source and target version are 1.8. so i have removed the jdk 1.7 reference.

Reference:
<plugin>
	<groupId>org.apache.maven.plugins</groupId>
	<artifactId>maven-compiler-plugin</artifactId>
	<version>3.6.0</version>
	<configuration>
		<source>1.8</source>
		<target>1.8</target>
		<encoding>UTF-8</encoding>
	</configuration>
</plugin>

Can you please let me know shall we add the  "openjdk7" and "oraclejdk7"?

Reply:
That looks okay to me. Just surprised that the .travis.yml file still contained JDK7 when it was compiled for JDK8.

Reply:
That's explained - the patch was originally for version 4.28, but version4 has moved to 4.5.0.